app/controllers/providers_controller.rb

```diff
@@ -177,6 +183,22 @@ def providers
     @providers ||= Provider.where(recruitment_cycle_year: Settings.current_cycle)
   end
 
+  def redirect_to_new_publish_url_about_provider_recruitment_cycle_path
+    redirect_to new_publish_url(about_provider_recruitment_cycle_path(@provider.provider_code, @provider.recruitment_cycle_year))
+  end
+
+  def redirect_to_new_publish_url_contact_provider_recruitment_cycle_path
+    redirect_to new_publish_url(contact_provider_recruitment_cycle_path(@provider.provider_code, @provider.recruitment_cycle_year))
+  end
+
+  def redirect_to_new_publish_url_details_provider_recruitment_cycle_path
+    redirect_to new_publish_url(details_provider_recruitment_cycle_path(@provider.provider_code, @provider.recruitment_cycle_year))
+  end
+
+  def redirect_to_new_publish_url_show_provider_recruitment_cycle_path
+    redirect_to new_publish_url(provider_recruitment_cycle_path(@provider.provider_code, @provider.recruitment_cycle_year))
+  end
+
```

Comment:
Does something like this work?

A before action in this controller:
`before_action :redirect_to_new_publish_equivalent, only: [:details, :contact, :about, :update]`

With the method definition being:
```
def redirect_to_new_publish_equivalent
  redirect_to new_publish_url(request.current_path)
end
```

Reply:
Actually that could be added to NewPublishHelper and accept the `request` object as an arg. Could then use that wherever needed as we migrate.

Reply:
done